syncd/scripts/syncd_init_common.sh

```diff
@@ -119,74 +119,175 @@ config_syncd_cisco_8000()
     fi
 }
 
-config_syncd_bcm()
+function merge_config_bcm_files()
 {
+    to_file=$1
+    from_file=$2
+    message=$3
+    override=false
+    echo "" >> $to_file
+    echo "# Start of $message" >> $to_file
+    while read line
+    do
+        line=$( echo $line | xargs )
+        if [ ! -z "$line" ];then
+            if [ "${line::1}" == '#' ];then
+                echo $line >> $to_file
+            elif [ "$line" == "[Overwrite Section]" ];then
+                override=true
+                echo "# $line" >> $to_file
+                echo "Merge properties with override $override"
+            else
+                sedline=${line%=*}
+                if grep -q $sedline $to_file ;then
```

Comment:
@geans-pin If there is "a=1" in overwrite section and "aa=1" in to_file, the sedline is a, but grep -q $sedline $to_file, still return true for this case, right? Will aa=1 be overwritten by a=1?

Reply:
In the config property, we grep for the property name. There is unlikely to have the a and aa case. 
The property name will be a meaningful naming prefix.